packages/api-server/package.json

```diff
@@ -4,12 +4,11 @@
   "description": "dummy package",
   "private": true,
   "scripts": {
-    "prepare": "cmake -B build && cmake --build build",
-    "test": "python3 -m pipenv run python -m unittest",
-    "test:cov": "python3 -m pipenv run python -m coverage run -m unittest && TEST_COVERAGE=1 npm run test:integration",
-    "test:integration": "python3 -m pipenv run python -m unittest discover -s tests",
+    "prepare": "",
```

Comment:
Should we remove this empty prepare?

Reply:
i think we can leave it here, npm looks at the prepare script to decide if it should install devdependencies